src/main/kotlin/racingcar/Moveable.kt

```diff
@@ -0,0 +1,24 @@
+package racingcar
+
+import java.util.concurrent.ThreadLocalRandom
+
+interface Moveable {
+    fun moving(): Boolean
+
+    class Always : Moveable {
+        override fun moving() = true
+    }
+
+    class Random : Moveable {
+        private val intRandom = IntRandom.Smart(ThreadLocalRandom.current(), BOUND_MAX)
```

Comment:
코틀린의 `Random.Default`를 사용하면 어떨까요?

```suggestion
        private val intRandom = IntRandom.Smart(kotlin.random.Random, BOUND_MAX)
```

Reply:
조금 다르게 사용했어요.
https://github.com/next-step/kotlin-racingcar/pull/143#discussion_r584546223